### DISCUSSION TITLE
PocketBase on Digital Ocean - deploying on pb_public folder, please help.

### DISCUSSION THREAD

**Discussion by Mashiane** [pocketbase/pocketbase#3358] [Q&A] [answered]
Hi there

I managed to create a droplet with PocketBase using this 1 click approach.
https://marketplace.digitalocean.com/apps/pocketbase

I selected the SSH key option

I am able to access the Digital Ocean via winscp using root username, the directory listing is showing. Wow.

I know nothing about linux. I want to create the pb_public folder, deploy my site there which will access the pocketbase instance running. I dont know where to find the folder where pocketbase is installed in and then create the pb_public folder.

Can someone advise please.

Thank you so much!

**Comment by ganigeorgiev** [comment, marked as answer]
Note that the 1-click DO app is not an official image and you may get better support by submitting a request in the related image script repo - https://github.com/scott/pocketbase-do-packer.

From a brief look at the [service file of the image](https://github.com/scott/pocketbase-do-packer/blob/b2e2c34f538d902801f0fa92fc736614729a1f6c/files/pocketbase.service#L14C32-L14C32), the directory of the executable seems to be `/opt/pocketbase/`.

With that said, you may also find it easier to deploy PocketBase on a clean/empty DO droplet following the [Minimal setup guide](https://pocketbase.io/docs/going-to-production/#minimal-setup). Practically you need to upload only the binary for your platform and start it (either directly or in the provided sample service file). [Hetzner](https://www.hetzner.com/cloud) is also a good and cheaper alternative.
